presto-main/src/test/java/com/facebook/presto/operator/TestGroupByHash.java

```diff
@@ -293,4 +299,90 @@ else if (type == BIGINT) {
         long[] expected = IntStream.range(0, actual.length).mapToLong(i -> entryBytes * expectedSize << (i + 1)).toArray();
         assertEquals(actual, expected);
     }
+
+    @Test(dataProvider = "dataType")
+    public void testPageYield(Type type)
```

Comment:
What does `PageYield` mean? I think `MemoryReservationYield` and `MemoryReservationYieldWithDictionary` are better names.